Description: Allow tagsoup 0.13
Author: Colin Watson <cjwatson@debian.org>
Forwarded: https://lists.debian.org/debian-haskell/2014/01/msg00002.html
Last-Update: 2014-01-01

Index: b/download-curl.cabal
===================================================================
--- a/download-curl.cabal
+++ b/download-curl.cabal
@@ -56,4 +56,4 @@
         build-depends:   base >= 3 && < 6, bytestring
     else
         build-depends:   base > 3 && < 4
-    build-depends: curl, tagsoup >= 0.8 && < 0.13, feed, xml
+    build-depends: curl, tagsoup >= 0.8 && < 0.14, feed, xml
